src/RBLstream.cc

```diff
@@ -364,6 +366,28 @@ DEF("prim-print",obPrint, 1, MaxArgs)
 }
 
 
+DEF("ostream-log-time",obLogTime, 1, 1)
+{
+    CHECK(0, Ostream, strm);
+    if (strm->stream) {
+        char buf[128];
+
+        time_t rawtime;
+        struct tm * timeinfo;
+        time (&rawtime);
+        timeinfo = localtime(&rawtime);
+        strftime(buf, sizeof(buf),"%Y-%m-%d %I:%M:%S\n",timeinfo);
+
+        if (errno = fprintf(strm->stream, "%s", buf))
+            return FIXNUM(-errno);
+        else
+            return NIV;
+    } else {
+        return PRIM_ERROR("cannot print on closed ostream");
+    }
+}
+
+
```

Comment:
leave a single line, not two

Reply:
Two seems consistent with the rest of the file.

Reply:
You're right, sorry; I just saw a nearby one was one line and assumed.